.travis.yml

```diff
@@ -13,13 +13,21 @@ env:
 install:
   - travis_retry make dependencies
 
-script:
-  - make install-dev-deps
-  - make check-style
-  - make test
+stages:
```

Comment:
We shouldn't have stages because the time required to boot another job is >> the check/test time.